src/market.rs

```diff
@@ -286,6 +289,26 @@ impl GenoaMarket {
         })
     }
 
+    pub fn update_news(&mut self) {
+        // Updating news by introducing noise and maintaining the value 
+        // between -1 and 1 (using the Sigmoid function).
+        self.news_indicator =
+            1. / (1. + (-self.news_indicator + rand::thread_rng().gen_range(-2.5..2.5)).exp());
+    }
+
+    pub fn get_news(&self) -> f32 {
+        // A simple value that indicates the status of a market.
+        self.news_indicator
+    }
+
+    pub fn get_markup(&self) -> f32 {
+        // Returns profit since last timestep as a percentage.
+        let history_len = self.price_history.len();
+        (self.price_history[history_len] - self.price_history[history_len - 1])
+            / self.price_history[history_len - 1]
+            * 100.
+    }
```

Comment:
This function will be incredible erratic, since the prices vary a lot. It is a bad predictor of the price.

Reply:
An alternative would be to measure the mark-up over a certain time frame, which should be less volatile. Either way, this value is only used as an indicator for market increase/decrease. 

I have removed the `* 100` to ensure that the value is between 0 and 1 in the decision-making function.